attendance/API_HR/c-hr.php

```diff
@@ -6170,14 +6170,20 @@ public static function API_getEmployeesLeavesStats( $year, $month ){
         $r_error = 0;
         $r_data = array();
         $stats = array();
-        $return = array();
+        $return = array();        
+        $weekendsOfMonth = self::getWeekendsOfMonth( $year, $month );
         $monthly_leaves = self::getLeavesForYearMonth( $year, $month );
```

Comment:
is it is of any use?

Reply:
this is using for getting all leaves data for a month, also use for creating stats data